layout/wmWindow/sheet.tsx

```
@@ -1,21 +1,149 @@
+        value: config.checked ?? false,
+      };
+    } else if (config._type === "string") {
+      // Handle string values
+      result[key] = {
+        value: config.value ?? "",
+      };
+    } else if (config._type === "computed") {
+      // Handle computed values - compute the formula result
+      const computedValue = computeFormula(config.formula, config.vars);
+      // Clamp the value between min and max if they exist
+      const min = config.min ?? -Infinity;
```

Biome

[error] Use Number.NEGATIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.NEGATIVE_INFINITY instead.


(lint/style/useNumberNamespace)